modules/classic_ui/pages/files/public_link_shares.adoc

```diff
@@ -53,5 +53,5 @@ Now that the share is created, you can:
 | Allows users to create xref:files/webgui/sharing.adoc#creating-drop-folders[a drop folder], which can receive files from multiple recipients without revealing the contents of the folder.
 
 | Password | Sets a password for protecting the link.
-| Expiration | Sets an expiry date for the link. The public link expires at the end of the specified day.
+| Expiration | Sets an expiry date for the link. The public link expires after the configured date as soon as the `OCA\Files_Sharing\ExpireSharesJob` job will run.
```

Comment:
PR https://github.com/owncloud/core/pull/38775 fixed this behavior, released in ownCloud core 10.8.0.
The ExpireSharesJob now correctly only removes shares that have an expiration date of yesterday or before. Shares that expire on the current day are left in-place, because they need to work until the very end of the day.